client/src/App.js

```diff
@@ -1,12 +1,51 @@
 import React from 'react';
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
 import './App.css';
+import Navbar from './navbar.js';
+import Projects from './projects.js';
+import Homepage from './homepage.js';
+import Work from './work.js';
+import Hobbies from './hobbies.js';
+import About from './about.js';
+import Footer from './footer.js';
+//import all .js pages
 
 function App() {
-  return (
-    <div className="App">
-      <h1>Hello portfolio!</h1>
-    </div>
+
+  return ( //import page links into navbar & footer. all accessible on all pages
+    <Router>
+      <div class="Navbar">
+        <Link to='./homepage.js'>Home Page</Link>
+        <Link to='./work.js'>Work</Link>
+        <Link to='./about.js'>About</Link>
+        <Link to='./projects.js'>Projects</Link>
+        <Link to='./hobbies.js'>Hobbies</Link>
+      </div>
+      <Switch> 
+        <Route exact path="/homepage.js">
+          <Homepage />
```

Comment:
for `exact path=` we want it to always = '/' rather then the homepage.js. We can tell it to render the `homepage.js` on going to the `exact path='/'`. This will make it so that when your page is first loaded up it will show your homepage.